examples/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# add_subdirectory(grpc) TODO
```

Comment:
Instead of using `grpc` as a subdirectory consider using it via `externalproject_add` like you do for Abseil.  That way you do not get their targets in your namespace, **and** you would be testing a configuration that more closely resembles what you get via `find_package()`.